### PR TITLE
use nvm() wrapper in jenkins for android builds

### PR DESCRIPTION
**Pull request by jakubgs**
Android builds seem to be the only ones not using this `nvm()` wrapper for correct NodeJS version.

### CODE REVIEW

File: ci/Jenkinsfile.android [hunk 1]
```diff
@@ -24,6 +24,7 @@ pipeline {
 
   environment {
     BUILD_PLATFORM = 'android'
+    NODE_VERSION = 'v10.14.1'
```
Comment (pedropombeiro) [inline ci/Jenkinsfile.android]: This should be `10.12.0`, right?

**Review by pedropombeiro — CHANGES_REQUESTED** [review]

File: ci/Jenkinsfile.android (hunk 1 above)
Comment (pedropombeiro) [reply ci/Jenkinsfile.android]: would be cool if we could read the version from `.TOOLVERSIONS` :-)

Comment (jakubgs) [reply ci/Jenkinsfile.android]: Oh, is that change already merged? Then i can try that.

**Review by pedropombeiro — APPROVED** [review]
Awesome, thanks Jakub!

**Review by mandrigin — APPROVED** [review]

**Pull request merged**